### DISCUSSION TITLE
How to set bot presense

### DISCUSSION THREAD

**Discussion by JustAnOpossum** [diamondburned/arikawa#320] [Help] [answered]
In V1 I used to be able to update the presence of the bot using UpdateStatus, is there anyway to update the presence of the bot in V3 since I couldn't find a corresponding method to update it or a way to update it. Thanks!

**Comment by diamondburned** [comment, marked as answer]
v3 now uses something like `.Gateway().Send(&gateway.UpdateStatusCommand{})`. You can also change the `gateway.DefaultIdentity` global variable or change the identity data by passing a custom gateway to the session.

**Comment by JustAnOpossum** [comment]
Yup it's `session.Gateway().Send(ctx, &gateway.UpdatePresenceCommand{))` Thanks!
